app/views/components/fields/FieldContentFormatter.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.components.fields
+
+import java.time.format.DateTimeFormatter
+import java.time.{Instant, ZoneId}
+
+object FieldContentFormatter {
+
+  def dateTime(dateTime: Instant): String =
+    DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm").withZone(ZoneId.systemDefault()).format(dateTime)
```

Comment:
`DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm").withZone(ZoneId.systemDefault())` could be moved to `val` in this object.

This would not build and parse formatter each request

Reply:
:+1: